extensions/amp-ad-network-nws-impl/0.1/test/test-nws-a4a-config.js

```diff
@@ -29,6 +30,8 @@ describes.fakeWin('amp-ad-network-nws-impl', {amp: true}, (env) => {
     });
     doc.body.appendChild(element);
     impl = new AmpAdNetworkNwsImpl(element);
```

Comment:
unit tests appear to be failing instantiation here, lmk if you want help looking at the error.

Reply:
Yeah, the error is a bit cryptic.
I was seeing it locally too, although now it seems to be running with no failures (`gulp pr-check`)

Reply:
Looks to be still failing (https://travis-ci.com/github/ampproject/amphtml/jobs/434239338). I don't think pr check will run unit tests? If not I normally do `gulp unit --nobuild --local_changes` or `gulp unit --files=xxx`, you can also add `--watch`

Reply:
Unfortunately, I can't seem to work out the reason for the error.
It gets thrown in the parent class constructor, and can't spot the difference with the doubleclick implementation.
Perhaps a non-obvious import is missing.

Reply:
sorted this out offline.